.github/workflows/snyk.yml

```diff
@@ -0,0 +1,67 @@
+name: "Snyk"
+
+on:
+  # Please uncomment the push event for testing purposes
+  #push:
+  #  branches: [main]
+  # Run the job at the end of every day
+  schedule:
+    - cron: "0 0 * * *"
+
+env:
+  # Please check how to create an access token here https://docs.snyk.io/tutorials/amazon-web-services/aws-code-suite/snyk-security/create-account-and-obtain-a-token
+  SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
+  DEFAULT_JDK_VERSION: 11
+
+jobs:
+  quarkus:
+    name: Quarkus
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - uses: actions/setup-java@v1
+        with:
+          java-version: "${{ env.DEFAULT_JDK_VERSION }}"
+      
+      - name: Build Keycloak
+        run: mvn clean install -nsu -B -e -Pdistribution -Dmaven.test.skip -DskipQuarkus -DskipTestsuite -DskipExamples -DskipTests
+
+      - uses: snyk/actions/setup@master
+      - name: Check for vulnerabilities
+        run: snyk test --policy-path=.snyk --all-projects --prune-repeated-subdependencies --exclude=tests --sarif-file-output=quarkus-report.sarif quarkus
+        continue-on-error: true
+        env:
+          SNYK_TOKEN: "${{ env.SNYK_TOKEN }}"
+
+      - name: Upload scanner results to GitHub
+        uses: github/codeql-action/upload-sarif@v1
+        with:
+          sarif_file: quarkus-report.sarif
+
+  operator:
+    name: Operator
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - uses: actions/setup-java@v1
+        with:
+          java-version: "${{ env.DEFAULT_JDK_VERSION }}"
+      
+      - name: Build Keycloak
+        run: mvn clean install -nsu -B -e -Pdistribution -Dmaven.test.skip -DskipQuarkus -DskipTestsuite -DskipExamples -DskipTests
```

Comment:
The operator module is skipped by default. This won't build it.

Reply:
@vmuzikar could you please elaborate more on this? Unless something has changed, this will build, providing all the necessary artifacts to the Snyk analysis.